Funcky.Money.Test/MoneyTest.cs

```diff
@@ -472,6 +457,32 @@ public void RoundingStrategiesMustBeInitializedWithAValidPrecision()
         Assert.Throws<InvalidPrecisionException>(() => _ = RoundingStrategy.RoundWithAwayFromZero(0.0m));
     }
 
+    [Fact]
+    public void WeCanCalculateADimensionlessFactorByDividingAMoneyByAnother()
+    {
+        Assert.Equal(2.5m, Money.CHF(5) / Money.CHF(2));
+        Assert.Equal(0.75m, Money.USD(3).Divide(Money.USD(4)));
+    }
+
+    [Fact]
+    public void DividingTwoMoneysOnlyWorksIfTheyAreOfTheSameCurrency()
+    {
+        Assert.ThrowsAny<MissingExchangeRateException>(() => Money.CHF(5) / Money.USD(2));
+    }
+
+    [Fact]
+    public void DividingTwoMoneysWithDifferentCurrenciesNeedAnEvaluationContext()
```

Comment:
Other tests contain "monies", so using moneys here seems inconsistent

Reply:
There was only one monies, but seems like moneys is more standard: https://writingexplained.org/monies-vs-moneys-difference. (So I removed the monies) Fixed in: 15edde265e4d73087133b79bd372c98656f2cec0